.github/workflows/regression-test-docker-image.yml

```diff
@@ -0,0 +1,13 @@
+name: "Regression Test - Docker Image"
+on:
+  workflow_dispatch:
+  push:
+    paths:
+      - '**regression-test-docker-image.yml'
```

Comment:
@krompaco if we can publish our new build it would be great if it changed at least also when `package.json` is changed also.

Reply:
and on software-full.json change, happens 2 times every month.
(contains CVE and version updates for supported software)

Reply:
I've updated the paths now. See other thread on publishing.